### ISSUE TITLE
Testnet user feedback loop

### ISSUE THREAD

**Issue by huisq** [Art3misOracle/website#6]
Add entrypoint for testnet phase user feedback. 
Simplistic and minimal, so to not interfere and affect the design of the website. 
eg: google form.

**Comment by OWK50GA** [comment]
Hi, @huisq
I'm a Frontend Web Developer adept at writing React and Next
I can handle this.
ETA - 1 day

**Comment by NueloSE** [comment]
Can I attempt this issue?

**Comment by Tzienom** [comment]
Hi. I would love to attempt this issue.

**Comment by GoSTEAN** [comment]
I’d love to work on this task.

**Comment by Judah04** [comment]
Hi, @huisq 
I'd love to work on this.

**Comment by mimisavage** [comment]
Can I work on this, please?

**Comment by Ndifreke000** [comment]
Can I contribute to this one?

**Comment by Supa-mega** [comment]
Can I take this from here?

**Comment by Luluameh** [comment]
May I be assigned to this?
Estimated Completion Time: 2 days

**Comment by ShantelPeters** [comment]
Mind if I try this one?

**Comment by akintewe** [comment]
Hi, I'm interested in taking on the this task. I am a frontend/ UIUX developer and i have experience in working with detailed designs. I've reviewed the requirements and I'm confident I can deliver. I'd love to discuss further and get started. Looking forward to hearing from you. ETA 8hours :)

**Comment by Michaelkingsdev** [comment]
Hello @huisq 
Kindly assign this task to me. i will deliver in 3 days. Thanks

**Comment by Jagadeeshftw** [comment]
Hello, I’d love to work on this issue as I have strong experience in developing secure and efficient smart contracts. Please assign this to me, and I’ll ensure to deliver a robust and high-quality PR within 12-24 hours.

**Comment by ryzen-xp** [comment]
Can I contribute to this one?

**Comment by BrunoAmbricca** [comment]
I would like to work on this issue

**Comment by Benjtalkshow** [comment]
I am a Full Stack Developer with a strong background in blockchain and extensive experience in Next.js/React, TypeScript, and Rust. I’ve made over 49 contributions to over 17 projects in the OnlyDust ecosystem, solving complex issues and delivering efficient, scalable solutions.

I can handle this task.
ETA is 3days.

**Comment by tasneemtoolba** [comment]
I would like to gain this experience!, Let me handle this issue!

**Comment by caxtonacollins** [comment]
Could I be assigned to this?

**Comment by josephpdf** [comment]
Hi @huisq 
I'm Joseph Poveda, a member of the Dojo Coding community, I have experience in different programming languages ​​and have contributed to different OSS projects.
To solve the issue I propose
1. **Define the Entry Point**:
   - Add a non-intrusive button or link labeled "Testnet Feedback" in a prominent location on the website.
2. **Integrate Feedback Form**:
   - Embed or link to a Google Form tailored for gathering user feedback.
3. **Design Considerations**:
   - Ensure the button or link aligns with the site's current design without disrupting the user experience.
4. **Testing**:
   - Verify functionality, accessibility, and visual integration.
5. **Deploy and Monitor**:
   - Implement the feature, then monitor feedback for improvements.

**Comment by DuendexCR** [comment]
I'd like to handle this task.

**Comment by huisq** [comment]
@akintewe 
additional details:
1. simple and elegant design matching with current website design
2. questions to ask users for feedback:
- rate your experience from 1-10 (10 being best)
- how satisfied are you with your reading
- how much are you willing to pay for a reading
- will you like to use art3mis oracle again on mainnet
- open feedback

Feel free to suggest other questions you think may be related or help with the process

**Comment by akintewe** [comment]
Ok thank you

**Comment by huisq** [comment]
@josephpdf are you working on this?

**Comment by josephpdf** [comment]
> @josephpdf are you working on this?

Yes @huisq, but I'm having trouble installing the aptos' dependencies

**Comment by huisq** [comment]
@josephpdf dont actually need that. use a google form or typeform will do
